doc-content/shared-kie-docs/src/main/asciidoc/Workbench/SecurityManagement/business-central-settings-changing-permissions-proc.adoc

```diff
@@ -0,0 +1,31 @@
+[id='business-central-settings-changing-permissions-proc_{context}']
+= Changing permissions for groups and roles in {CENTRAL}
+
+In {CENTRAL}, you cannot change permissions for an individual user. However, you can change permissions for groups and roles. The changed permissions apply to users that have the role or belong to the group that you changed.
+
+[NOTE]
+====
+Any changes you make to roles or groups affect all of the users associated with that role or group.
+====
+
+.Procedure
+. To access the *Security management* page in {CENTRAL}, select the *Admin* icon in the top-right corner of the screen.
+. Click *Roles*, *Groups*, or *Users* on the {CENTRAL} *Settings* page.
++
+Depending on which icon you click, the *Security management* page opens for the respective tab.
```

Comment:
The *Security management* page opens on the tab for the icon that you clicked.

Reply:
Done